genkit-tools/src/commands/eval-flow.ts

```diff
@@ -178,25 +187,37 @@ export const evalFlowRun = new Command('eval:flow')
     }
   );
 
-async function readInputs(data: string, filePath: string): Promise<any[]> {
+async function readInputs(
+  data: string,
+  filePath: string
+): Promise<EvalFlowInput> {
   const parsedData = JSON.parse(
     data ? data : await readFile(filePath!, 'utf8')
   );
   if (Array.isArray(parsedData)) {
     return parsedData as any[];
   }
 
-  return [parsedData];
+  try {
+    return EvalFlowInputSchema.parse(parsedData);
+  } catch (e) {
+    throw new Error(
+      `Error parsing the input. Please provide an array of inputs for the flow or a ${EVAL_FLOW_SCHEMA} object`
```

Comment:
Should we also include the error details in the message so they know why it failed?

Reply:
Done.